build.gradle

```diff
@@ -188,6 +188,7 @@ dependencies {
     compile "com.sigopt:sigopt-java:4.9.0"
 
     compile("com.uber:h3:3.4.1")
+    compile("com.github.LBNL-UCB-STI:jsprit-wrapper:v0.5.1")
```

Comment:
Does this part need reviewed (as I believe it is a new codebase) - or did somebody else already do that.

Reply:
This part is not reviewed yet. Could you please review it https://github.com/LBNL-UCB-STI/jsprit-wrapper ?